addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4302,6 +4418,43 @@ registry.sizing = SnippetOptionWidget.extend({
         // updateUI is called when the target is changed
         this._onResize();
     },
+    /**
+     * @override
+     */
+    async updateUIVisibility() {
```

Comment:
I don't see the call to `_super` here, you should, even if nothing is broken because of it